woocommerce/woocommerce-ios
# [Mobile Payments] Show About Tap to Pay from Setup flow's learn more link

Closes: #10902


## Description


Since we've added a native `About Tap to Pay on iPhone` screen for TTPUK, we decided update our Set up flow to open that, rather than the web view with the docs for TTP.

This PR changes the `Learn More` link/button to present the About screen, and updates the About screen to match designs when there's no button present.

## Testing instructions


1. Delete the app, then build and run from Xcode
2. Log in, and select a US or UK-based WooPayments store
3. Navigate to `Menu > Payments > Set up Tap to Pay on iPhone`
4. Tap `Learn More`
5. Observe that the `About Tap to Pay on iPhone` screen is shown, not a web view.
6. Note that there is a `Done` button in the nav bar, which takes you back, and no `Set up Tap to Pay` CTA visible.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/bf27c456-13ac-41ab-b5be-84070c903474



---
